example_files/example_project_with_electric_network.json

```diff
@@ -2065,52 +1800,8 @@
           ]
         ]
       }
-    },
-    {
-      "type": "Feature",
-      "properties": {
-        "id": "c8181999-1d82-4f4f-b0d7-7affddf06378",
-        "type": "ElectricalJunction",
-        "DSId": "1b4b5dcf-8d18-4954-b655-a93634542733"
-      },
-      "geometry": {
-        "type": "Point",
-        "coordinates": [
-          -78.85138750076295,
-          42.81573237349922
-        ]
-      }
-    },
-    {
-      "type": "Feature",
-      "properties": {
-        "id": "f57d88d3-925f-4d5d-b25c-b3b9475f1ea7",
-        "type": "ElectricalConnector",
-        "startJunctionId": "2796b96e-5bc0-4890-9bb0-bcf7b5baf9c7",
-        "endJunctionId": "c8181999-1d82-4f4f-b0d7-7affddf06378",
-        "total_length": 121,
-        "wires": [
-          "OH AL 2/0 A",
-          "OH AL 2/0 B",
-          "OH AL 2/0 C",
-          "OH AL 2/0 N"
-        ],
-        "connector_type": "Wire",
-        "name": "p10"
-      },
-      "geometry": {
-        "type": "LineString",
-        "coordinates": [
-          [
-            -78.85100126266484,
-            42.81555923181503
-          ],
-          [
-            -78.85138750076295,
-            42.81573237349922
-          ]
-        ]
-      }
     }
+    
+
```

Comment:
Do we need this extra spacing?

Reply:
I can't imagine we do. Thanks! I'll remove it.